app/code/Magento/Catalog/Controller/Adminhtml/Product/Save.php

```diff
@@ -270,31 +282,23 @@ protected function copyToStores($data, $productId)
     }
 
     /**
-     * Get categoryLinkManagement in a backward compatible way.
+     * Get CategoryLinkManagementInterface
      *
      * @return \Magento\Catalog\Api\CategoryLinkManagementInterface
      */
     private function getCategoryLinkManagement()
```

Comment:
Please, remove method and replace usages to `$this->categoryLinkManagement`

Reply:
Fixed, thank you!